index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/cf2fdf395d.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <nav class="navbar">
+        <a href="#about"><i class="fas fa-bars"></i></a>
+        <a class="active" href="#home"></a><i class="fas fa-search"></i></a>
+        <a href="#home">Space & Cosmos</a>
+        <a href="#home">New York Times</a>
+        <button type="submit">Subscribe</button>
+        <button type="submit">Sign up</button>
+    </nav>
+    <h1>Space Ripples Reveal Big Bang’s Smoking Gun</h1>
+    <img src="https://static01.nyt.com/images/2014/03/18/science/space/18cosmos/18cosmos-superJumbo.jpg?quality=90&auto=webp" alt="Italian Trulli">
+        <p>Alan Guth was one of the first physicists to hypothesize the existence of inflation, which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago</p>
+        <p>Rick Friedman for The New York Times</p>
+     <h2>By <a href="#top">Dennis Overbye</a></h2>   
+     <h3>March 17, 2014</h3>
+     <p>Sign up for Science Times: Get stories that capture the wonders of nature, the cosmos and the human body.</p>
+     <button>sign up</button>
+        <div class="middle-section">
+            <p>CAMBRIDGE, Mass. — One night late in 1979, an itinerant young physicist named Alan Guth, with a new son and a year’s appointment at Stanford, stayed up late with his notebook and equations, venturing far beyond the world of known physics.</p>
+            <p>He was trying to understand why there was no trace of some exotic particles that should have been created in the Big Bang. Instead he discovered what might have made the universe bang to begin with. A potential hitch in the presumed course of cosmic evolution could have infused space itself with a special energy that exerted a repulsive force, causing the universe to swell faster than the speed of light for a prodigiously violent instant.</p>
```

Comment:
Please, consider using semantic tags like <section> or <article> before submitting the code for review.